integration_tests/integration/link/requestLink.spec.ts

```diff
@@ -0,0 +1,86 @@
+import Page from '../../pages/page'
+import RequestLinkPage from '../../pages/link/requestLink'
+import EmailSentPage from '../../pages/link/emailSent'
+
+context('Request Link', () => {
```

Comment:
As discussed - maybe give the fluent style used in the PoC a try and see if you like it.

Reply:
👍